File: src/main/java/no/nav/pto_schema/enums/arena/SikkerhetstiltakType.java
package no.nav.pto_schema.enums.arena;

public enum SikkerhetstiltakType {
    FTUS("FTUS"), // Fysisk/telefonisk utestengelse
    FYUS("FYUS"), // Fysisk utestengelse
    TOAN("TOAN"), // To ansatte i samtale
    TFUS("TFUS"); // ?

    private final String kode;

    SikkerhetstiltakType(final String kode) {
        this.kode = kode;
    }

    @Override
    public String toString() {
        return kode;
    }
}

File: src/main/kotlin/no/nav/pto_schema/enums/arena/SikkerhetstiltakType.kt
package no.nav.pto_schema.enums.arena

enum class SikkerhetstiltakType(private val kode: String) {
    FTUS("FTUS"), // Fysisk/telefonisk utestengelse
    FYUS("FYUS"), // Fysisk utestengelse
    TOAN("TOAN"), // To ansatte i samtale
    TFUS("TFUS"); // ?

    override fun toString(): String = kode
}

